source/features/pull-request-hotkey.tsx

```diff
@@ -2,7 +2,7 @@ import select from 'select-dom';
 import features from '../libs/features';
 
 function init() {
-	const tabs = select.all('.tabnav-pr .tabnav-tab');
+	const tabs = select.all<HTMLElement>('.tabnav-pr .tabnav-tab');
```

Comment:
Used the generic `HTMLElement` as I have no idea what kind of element this is supposed to be. I would guess a `<div />`, but not certain.

Reply:
I'd stick to `HTMLElement`